pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -21,13 +21,19 @@
       }
     },
     "Microsoft.Bcl.AsyncInterfaces": {
+      "StableVersions": [
```

Comment:
I assume those changes are auto-generated?

Reply:
yes this is auto-generated using https://github.com/dotnet/corefx/blob/da147ece700c2502b91e57d99213feadcfb9c1f0/src/packages.builds#L29 target